src/components/oidc/OidcConfig.ts

```diff
@@ -33,6 +32,9 @@ export function getRolesOfUser(oidcUserInfo: object): string[] {
 
     if (availableRoleList.length === 0) {
         availableRoleList.push('user'); // default role when no roles are assigned.
+    } else {
+        const userNamespace: string = getMetadataByKey(oidcUserInfo, 'namespace');
+        console.log('namespace of user:', userNamespace);
```

Comment:
@baixinsui we are only logging the user namespace here? Do we really need to fetch the 'namespace'?

Reply:
The UI  need not to get 'namespace'. I  deleted related code.